.gitignore

```diff
@@ -1078,3 +1078,6 @@ hedera-node/hapi/checkouts.bin
 
 ### CI Pipeline Created Files
 gha-creds-*.json
+/platform-sdk/swirlds-config-processor/src/main/java/com/swirlds/config/processor/antlr/generated/*.interp
+/platform-sdk/swirlds-config-processor/src/main/java/com/swirlds/config/processor/antlr/generated/*.tokens
```

Comment:
```suggestion
**/antlr/generated/
!**/antlr/generated/*.java
```

Reply:
At the moment I want to have it really for that module. We do not know if that will be the default in future